docs/advanced-usage/access-control/kerberos/index.md

```diff
@@ -47,10 +47,22 @@ $ reana-client secrets-add --env CERN_USER=johndoe \
 
 ## Setting Kerberos requirement
 
-Set `kerberos: true` for the steps in need in the workflow specification.
-Please note that step's docker image (e.g `environment: 'cern/slc6-base'`)
-should have Kerberos client installed and you have to for the Kerberos
-authentication to work.
+If the workflow engine you are using needs Kerberos to parse and validate the
```

Comment:
**Setting Kerberos requirement for whole workflow**

(Comment: I think it may be useful to introduce two sections, how to specify krb5 for the whole workflow vs how to specify it only for some of its steps.  The explanation will be clearer that way. We don't have to introduce new subsections, only some title-line demarcation in bold would be useful.)

Reply:
> We don't have to introduce new subsections, only some title-line demarcation in bold would be useful.

I have introduced two subsections so that they show up in the table of contents on the right side of the page, but we can change them to be simple title in bold if you prefer. What do you think?

Reply:
I think it reads well :+1: